hack/install.sh

```diff
@@ -203,6 +169,7 @@ install_required_packages
 $SUDO /sbin/sysctl -w net.ipv4.conf.all.forwarding=1
 echo "net.ipv4.conf.all.forwarding=1" | $SUDO tee -a /etc/sysctl.conf
 
+install_arkade
 install_cni_plugins
 install_containerd
 install_faas_cli
```

Comment:
Why not use arkade to also download faas-cli?

Reply:
Done